contract/report/report_contract.xml

```diff
@@ -38,7 +36,7 @@
                                     <tbody>
                                         <tr t-foreach="o.recurring_invoice_line_ids" t-as="l">
                                             <td>
-                                                <span t-field="l.name"/>
+                                                <span t-field="l.name_on_contract"/>
```

Comment:
Field `name_on_contract` is filled in two cases:

- on creation of a line
- onchange product_id of a line

Shouldn't be calculated also for the existing "Contract Lines" that are present at the moment the module is installed?